modules/enabling-team-sync-oidc.adoc

```diff
@@ -0,0 +1,210 @@
+<2> Required. The registered OIDC client secret for this {productname} instance.
+<3>  Required. The address of the OIDC server that is being used for authentication. This URL should be such that a `GET` request to `<OIDC_SERVER>/.well-known/openid-configuration` returns the provider's configuration information. This configuration information is essential for the relying party (RP) to interact securely with the OpenID Connect provider and obtain necessary details for authentication and authorization processes.
+<4> Required. The name of the service that is being authenticated.
+<5> Required. The key name within the OIDC token payload that holds information about the user's group memberships. This field allows the authentication system to extract group membership information from the OIDC token so that it can be used with {productname}.
+<6> Required. Adds additional scopes that {productname} uses to communicate with the OIDC provider. Must include `'openid'`. Additional scopes are optional.
+<7> Required. Whether to allow for team membership to be synced from a backing group in the authentication engine.
+<8> Optional. If enabled, non-superusers can setup team synchronization.
+
+. Restart your {productname} registry. 
+
+[id="setting-up-quay-team-sync"]
+== Setting up your {productname} deployment for team synchronization
+
+. Log in to your {productname} registry via your OIDC provider.
+
+. On the {productname} v2 UI dashboard, click *Create Organization*.
+
+. Enter and Organization name, for example, `test-org`.
+
+. Click the name of the Organization.
+
+. In the navigation pane, click *Teams and membership*. 
+
+. Click *Create new team* and enter a name, for example, `testteam`.
+
+. On the *Create team* pop-up:
+
+.. Optional. Add this team to a repository.
+.. Add a team member, for example, `user1`, by typing in the user's account name.
+.. Add a robot account to this team. This page provides the option to create a robot account.
+
+. Click *Next*.
+
+. On the *Review and Finish* page, review the information that you have provided and click *Review and Finish*. 
+
+. To enable team synchronization for your {productname} OIDC deployment, click *Enable Directory Sync* on the *Teams and membership* page. Note the message in the popup:
++
+[WARNING]
+====
+Please note that once team syncing is enabled, the membership of users who are already part of the team will be revoked. OIDC group will be the single source of truth. This is a non-reversible action. Team's user membership from within Quay will be ready-only.
+====
+
+. In the popup box, enter the name of the group to sync membership with. Then, click *Enable Sync*. 
+
+. You are returned to the *Teams and membership* page. Note that users of this team are removed and are re-added upon logging back in. At this stage, only the robot account is still part of the team.
++
+A banner at the top of the page confirms that the team is synced: 
```

Comment:
QE didn't see this banner in testing env, need double confirm with @Sunandadadi

Reply:
Is this the accordion @Sunandadadi ?

Reply:
No, this the banner in blue (above accordion):

![Screenshot 2024-03-19 at 10 40 21 AM](https://github.com/quay/quay-docs/assets/11522230/6f8854d4-21d9-4999-a979-425f611b1e88)